applications/fishing-map/src/features/map/map.slice.ts

```diff
@@ -116,11 +116,11 @@ export const fetch4WingInteractionThunk = createAsyncThunk<
     // activity heatmaps use the same time intervals, This will need to be revised in case we support interactivity on environment layers
     const start =
       mainFeature.temporalgrid?.interval === '10days'
```

Comment:
I actually think this condition should be removed, and vessel queries should always be made depending on the real 4w time range, not the one in timebar (even though in practice it only makes a difference for `10days` because of timebar 'stickiness' with day/hour)

Reply:
We used to have a bug with the `day` period where the value of `visibleStartFrame` didn't match the real one. But it seems to be fixed with these changes. Do you want me to do it?

Reply:
Did it, it's working fine